.github/workflows/cypress-update-snapshots.yml

```diff
@@ -0,0 +1,82 @@
+name: Cypress update snapshots
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      testname:
+        description: "Enter the test name to run (see e2e/specs/[testname].spec.js). For example: st_alert"
+        default: "st_alert"
+        type: "string"
+        required: true
+
+# Avoid duplicate workflows on same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ inputs.testname }}
```

Comment:
Another GHA caveat: the `inputs` context is not directly available in the workflow level concurrency statement - as noted in [this issue](https://github.com/orgs/community/discussions/35341).
But for `workflow_dispatch` triggers such as this, you can access inputs via the `event` payload:
`${{ github.event.inputs.testname }}`

Reply:
Ah, so that's why you had `github.event` in there. I was wondering about that. I should have just used your line verbatim, LOL!